faststreams.nimble

```diff
@@ -32,7 +32,7 @@ task test, "Run all tests":
   # TODO chronos backend uses nested waitFor which is not supported
   for backend in ["-d:asyncBackend=none"]:
     for threads in ["--threads:off", "--threads:on"]:
-      for mode in ["-d:debug", "-d:release", "-d:danger"]:
+      for mode in ["-d:debug", "-d:release", "-d:danger", "-d:useMalloc"]:
```

Comment:
```suggestion
      for mode in ["-d:debug", "-d:release", "-d:danger", "-d:useMalloc -d:noSignalHandler"]:
```

If you want to run with valgrind, araq recommended -d:noSignalHandler to me as well. but maybe not needed here. not sure.

Reply:
Well, this is run automatically from CI, and not under valgrind. So I'm not sure adding `-d:noSignalHandler` makes sense here. `-d:useMalloc` is useful because it can automatically catch some issues which where the usual pool allocators of ARC and ORC will seemingly avoid UAF's because they won't do the actual freeing.

A human running it can definitely add whatever flags they want.